model/converter/json/from_domain.go

```diff
@@ -122,6 +128,10 @@ func (fd fromDomain) convertKeyValues(keyValues model.KeyValues) []json.KeyValue
 			value = kv.Bool()
 		case model.Int64Type:
 			value = kv.Int64()
+			if kv.Int64() > js_limit_max || kv.Int64() < js_limit_min {
+				kv.VType = 0
```

Comment:
this needs to go into temp variable above the switch, must not change (in place) the input data

Reply:
I could not get why this change is required. If we move this if 
if kv.Int64() > jsMaxSafeInteger || kv.Int64() < jsMinSafeInteger  {
				kv.VType = model.StringType
                                value = fmt.Sprintf("%d", value)
			}
before switch,  the switch case model.StringType will be executed and the previous value will be overwriiten.

Reply:
You should not be changing the type at all, just the value to be a string.